.github/scripts/update-tags.sh

```diff
@@ -20,6 +20,11 @@ if ! command -v yq &> /dev/null; then
   exit 1
 fi
 
+if ! command -v npm &> /dev/null; then
```

Comment:
npm is not used by this script, so this should  be removed.

Reply:
I think the missing component is some kind of npm install action around here:
https://github.com/spiffe/helm-charts/blob/main/.github/workflows/helm-chart-ci.yaml#L36

Reply:
Hi @kfox1111 , the script uses `helm-docs.sh` which requires `npm` so have added the check here. The github [runner](https://github.com/actions/runner-images/blob/releases/ubuntu22/20230724/images/linux/Ubuntu2204-Readme.md) we are using comes pre-installed with the latest stable version of `npm` so it looks like we will be good without installing it explicitly.

Reply:
Ah. gotcha. Thanks.